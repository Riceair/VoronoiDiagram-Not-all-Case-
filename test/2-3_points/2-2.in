#雙點測試 水平
2
200 200
400 200