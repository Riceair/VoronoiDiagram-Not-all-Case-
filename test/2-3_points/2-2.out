P 200 200
P 400 200
E 300 -125800 300 126200
